client_src/js/utils/time.util.js

```diff
@@ -21,3 +21,11 @@ export function getLastThirtyDaysText() {
 
   return lastThirtyDaysText;
 }
+
+/**
+ * Format a date string to human readable date.
+ * @returns {String} Human readable date string
+ */
+export function formatDateStringHumanReadable(date) {
```

Comment:
Shorten formatter as this function name is longer than the normal call

Reply:
I wrote the wrapper as I'd originally planned to use the formatter but decided that should probably be done in another PR. I'll revert this as it's only used in printer connection log. As you say is unneeded.